docs/iidm/model/shuntCompensator.md

```diff
@@ -12,6 +12,10 @@ The `com.powsybl.iidm.network.ShuntCompensator` interface is used to model a shu
 | bPerSection | double | S | yes | - | The Positive sequence shunt (charging) susceptance per section |
 | MaximumSectionCount| integer | int | yes | - | The maximum number of sections that may be switched on |
 | CurrentSectionCount | integer | int | yes | - | The current number of section that may be switched on |
+| RegulatingTerminal | [`Terminal`](terminal.md) | - | no | The shunt compensator's terminal | The terminal used for regulation |
+| TargetV | double | kV | only if `VoltageRegulatorOn` is set to `true` | - |  The voltage target |
+| TargetDeadband | double | kV | only if `VoltageRegulatorOn` is set to `true` | - | The deadband used to avoid excessive update of controls |
+| VoltageRegulatorOn | boolean | - | no | false | The voltage regulating status |
 
 ## Section
 A section of a shunt compensator is an individual capacitor or reactor.
```

Comment:
Can you add maybe a sentence to explain that this regulation could be used or not by power flows to model a human action to increase or decrease a local volatge plan ?

Reply:
Done (below)